### PR TITLE
[feat] Display bookmarks list

### PR DESCRIPTION
**Pull request by arsengit**
- Display `bookmarks list` in the `Home` page

### CODE REVIEW

**Review by rubenaprikyan — APPROVED** [review]
Lgtm!

File: aim/web/ui/src/pages/Home/components/ExperimentsCard/ExperimentsCard.scss [hunk 1]
```diff
@@ -1,14 +1,26 @@
 @use 'src/styles/abstracts' as *;
 
 .ExperimentsCard {
-  padding: 1rem 1.5rem;
-  min-width: 550px;
+  border-top: $border-main;
+  padding: 1rem;
```
Comment (VkoHov) [inline aim/web/ui/src/pages/Home/components/ExperimentsCard/ExperimentsCard.scss:5]: Let's use $space-unit variable here

File: aim/web/ui/src/pages/Home/components/ExperimentsCard/useExperimentsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      experimentsEngine.experimentsState.destroy();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  // memoized table data
+  const tableData = React.useMemo(() => {
+    if (experimentsStore.data) {
+      return experimentsStore.data.map(
+        ({ name, archived, run_count }: any, index: number) => {
```
Comment (VkoHov) [inline aim/web/ui/src/pages/Home/components/ExperimentsCard/useExperimentsCard.tsx:30]: Can we set a type here

File: aim/web/ui/src/pages/Home/components/ExperimentsCard/useExperimentsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      },
+      {
+        dataKey: 'run_count',
+        key: 'run_count',
+        title: 'Runs',
+        flexGrow: 1,
+        width: '15%',
+        cellRenderer: ({ cellData }: any) => <p title={cellData}>{cellData}</p>,
+      },
+      // {
```
Comment (VkoHov) [inline aim/web/ui/src/pages/Home/components/ExperimentsCard/useExperimentsCard.tsx:134]: Remove the comment

**Review by VkoHov — APPROVED** [review]
lgtm!

**Pull request merged**